ui/src/criteria/concept.tsx

```diff
@@ -92,7 +92,7 @@ class _ implements CriteriaPlugin<Data> {
   }
 
   renderInline() {
-    return <ConceptInline data={this.data} />;
+    return <ConceptInline />;
```

Comment:
unrelated change?

Reply:
`props` became an unused variable with the removal of the concept in the inline display so I had to remove it entirely.